backend/nest-app/src/users/users.service.ts

```diff
@@ -1,33 +1,30 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { Injectable } from '@nestjs/common';
 import { MikroORM } from '@mikro-orm/core';
-import { CreateUserDto } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { User } from './users.entity';
+import { UserRepository } from './users.repository';
 
 @Injectable()
 export class UsersService {
-  constructor(private readonly orm: MikroORM) {}
-
-  create(createUserDto: CreateUserDto) {
-    return 'This action adds a new user';
-  }
+  constructor(
+    private readonly orm: MikroORM,
+    private readonly userRepository: UserRepository,
+  ) {}
 
   async getAllUsers(): Promise<User[]> {
-    const userRepository = this.orm.em.getRepository(User);
-    return userRepository.findAll();
+    return this.userRepository.findAllUsers();
   }
 
   async findOneById(id: number) {
```

Comment:
```suggestion
  async getUser(id: number) {
```
findOneById etc are conventions for the repository layer. Service layer is more human friendly and business model related.